Subject: [PATCH] cs416x: use upcase in config
X-BK-ChangeSetKey: akpm@osdl.org[torvalds]|ChangeSet|20050108213649|44654
Date:	Sat, 08 Jan 2005 23:09:19 +0000
Patch-mainline: yes
From: olh@suse.de


ChangeSet 1.2257, 2005/01/08 13:36:49-08:00, akpm@osdl.org

	[PATCH] cs416x: use upcase in config
	
	Kconfig symbols are always uppercase.
	
	Signed-off-by: Andrew Morton <akpm@osdl.org>
	Signed-off-by: Linus Torvalds <torvalds@osdl.org>



 Kconfig  |    2 +-
 Makefile |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)


diff -Nru a/drivers/input/gameport/Kconfig b/drivers/input/gameport/Kconfig
--- a/drivers/input/gameport/Kconfig	2005-01-08 15:06:35 -08:00
+++ b/drivers/input/gameport/Kconfig	2005-01-08 15:06:35 -08:00
@@ -84,7 +84,7 @@
 	tristate "ForteMedia FM801 gameport support"
 	depends on GAMEPORT
 
-config GAMEPORT_CS461x
+config GAMEPORT_CS461X
 	tristate "Crystal SoundFusion gameport support"
 	depends on GAMEPORT
 
diff -Nru a/drivers/input/gameport/Makefile b/drivers/input/gameport/Makefile
--- a/drivers/input/gameport/Makefile	2005-01-08 15:06:35 -08:00
+++ b/drivers/input/gameport/Makefile	2005-01-08 15:06:35 -08:00
@@ -5,7 +5,7 @@
 # Each configuration option enables a list of files.
 
 obj-$(CONFIG_GAMEPORT)		+= gameport.o
-obj-$(CONFIG_GAMEPORT_CS461x)	+= cs461x.o
+obj-$(CONFIG_GAMEPORT_CS461X)	+= cs461x.o
 obj-$(CONFIG_GAMEPORT_EMU10K1)	+= emu10k1-gp.o
 obj-$(CONFIG_GAMEPORT_FM801)	+= fm801-gp.o
 obj-$(CONFIG_GAMEPORT_L4)	+= lightning.o
-
To unsubscribe from this list: send the line "unsubscribe bk-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

